### DISCUSSION TITLE
Possible to install the pro version on a dev server?

### DISCUSSION THREAD

**Discussion by tfchristie** [netmix/radio-station#397] [General]
Just wondering - 
Rather than muck up our production web server, is there a way to install the pro version on our non-public facing development box to test things?  Like the player, for instance.  
Thanks!

**Comment by tonyzeoli** [comment]
The license key is already be set to utilize on various dev and staging
servers.

See:
https://freemius.com/help/documentation/selling-with-freemius/license-utilization/

You should already be good to go.

**Reply by tfchristie** [reply]
Awesome!  Thanks for the info.

**Comment by tonyzeoli** [comment]
This discussion is considered closed.
